src/widgets/block.rs

```diff
@@ -264,15 +421,16 @@ impl<'a> Block<'a> {
     ///
     /// ```
     /// // Draw a block nested within another block
-    /// use ratatui::{backend::TestBackend, buffer::Buffer, terminal::Terminal, widgets::{Block, Borders}};
+    /// use ratatui::{
+    ///     backend::TestBackend,
+    ///     buffer::Buffer,
+    ///     terminal::Terminal,
+    ///     widgets::{Block, Borders},
+    /// };
```

Comment:
This example needs to be shortened to just the interesting parts. E.g.:
```rust
/// # use ratatui::{prelude::*, widgets::Block};
/// # fn render_nested_block<B: Backend>(frame: &mut Frame) {
/// let outer = Block::default().title("Outer").borders(Borders::ALL);
/// let inner = Block::default().title("Inner").borders(Borders::ALL);
/// let outer_area = frame_size();
/// let inner_area = outer.inner(outer_area);
/// frame.render_widget(outer_block, outer_area);
/// frame.render_widget(inner_block, inner_area);
/// # }
```
renders:
```plain
┌Outer Block──┐
│┌Inner Block┐│
││           ││
│└───────────┘│
└─────────────┘
```

Reply:
I wanted to reduce it but didn't thought of using a hidden function. Anyway fixed it.